src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -338,14 +346,25 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
             {
                 trace::verbose(_X("Executing as a standalone app as per config file [%s]"), config_file.c_str());
                 pal::string_t impl_dir = get_directory(app_or_deps);
-                if (!library_exists_in_dir(impl_dir, LIBHOSTPOLICY_NAME, nullptr) && !probe_path.empty() && !deps_file.empty())
+                if (!library_exists_in_dir(impl_dir, LIBHOSTPOLICY_NAME, nullptr) && !probe_paths.empty() && !deps_file.empty())
```

Comment:
Isnt this the case of resolving hostpolicy from dev nuget cache? If so, then we should only be consulting the probe path for that cache and not the entire set.